editor/src/clj/editor/tile_map.clj

```diff
@@ -1207,20 +1242,29 @@
   (let [^Point3d screen-pos (:screen-pos action)]
     (case (:type action)
       :mouse-pressed
-      true
+      (do
+        (g/transact
+          (g/set-property self :start-palette-tile (g/node-value self :palette-tile evaluation-context)))
+        true)
```

Comment:
We don't need in `:palette`a state machine with different operations like we have in the `:editor`, because here we have only one mode: select